src/components/hub/ChatForm.js

```diff
@@ -72,6 +81,25 @@ export default function ChatForm() {
   }, [isTabInFocus, readMessage]);
 
   useEffect(() => {
+    if (selectedConversation?.is_encrypted) {
+      const isEncryptedUrlAuth = location.pathname.includes("/auth_encrypted");
```

Comment:
This can be moved outside of `useEffect`, e.g.

```
const isLockScreenPath = location.pathname.includes("/auth_encrypted");
```


and also can be used below

```
 {isLockScreenPath? (
        <LockScreen />
      )
```